sshutil/validate_key.go

```diff
@@ -22,7 +23,7 @@ func ValidatePrivateKey(privateKey []byte, username string, url string) (bool, e
 	if _, err = git.Clone(memory.NewStorage(), nil, &git.CloneOptions{
 		Auth:              SSHAuth,
 		URL:               url,
-		Progress:          os.Stdout,
+		Progress:          bytes.NewBuffer([]byte{}),
```

Comment:
Would add a progress indicator (https://github.com/bitrise-io/go-utils/tree/master/progress) while validation is running. The output could be printed out as a Debug level log.